architecture-building-systems/CityEnergyAnalyst
# I2204 temperature setpoints

this PR contains the following changes:
- Temperature set point / set back schedules are moved to the occupancy database. These schedules consist of temperature values or `OFF`.
- Temperatures are therefore removed from the `comfort.dbf`
- The occupancy model adds these temperature schedules to the schedule files according to the main use of the buildings.
- These schedules are directly used to control heating and cooling systems. The only other parameter controlling heating and cooling operation is the heating season and cooling season.

How to test:
- check if all schedules in the database have temperature values
- run the data-helper
- run the demand (for different cases)
- check the schedules output files
- check the demand output files
